packages/shared/src/utils/constants/contract.tsx

```diff
@@ -399,17 +406,17 @@ export const getSupportedContracts = is_high_low => ({
     },
 });
 
-export const getContractConfig = is_high_low => ({
+export const getContractConfig = (is_high_low: boolean) => ({
     ...getSupportedContracts(is_high_low),
     ...getUnsupportedContracts(),
 });
 
-export const getContractTypeDisplay = (type, is_high_low = false) => {
-    return getContractConfig(is_high_low)[type] ? getContractConfig(is_high_low)[type.toUpperCase()].name : '';
+export const getContractTypeDisplay = (type: TGetSupportedContracts, is_high_low = false) => {
```

Comment:
I think it's better to have consistency for the code typing. Both of these codes are the same:

`is_high_low: boolean` and `is_high_low = false`

I think it's better to use one of them in the code.

Reply:
Thank u @hamid-deriv for your revision, we don't need return type for `getContractTypeDisplay`.
About the `is_high_low`, it is boolean everywhere but in 2 functions we have a default value for it!

Reply:
@niloo-fs I didn't get the first part about the return type of the `getContractTypeDisplay`. Why is there no need to define the return type for it?

The boolean type is `false` as default and I think it's unnecessary to define it implicitly. So `is_high_low: boolean` is the same as `is_high_low = false`.

Reply:
About the first part the **returntype** of `getContractTypeDisplay` by default is `JSX.Element` and it is unnecessary to define that again

Reply:
About the `boolean` type I didn't use to define that before, but I saw in some of the components they defined types for these kind of props so I added the type for it!